memstore/scheduler.go

```diff
@@ -228,6 +228,12 @@ func (scheduler *schedulerImpl) Start() {
 }
 
 func (scheduler *schedulerImpl) executeJob(jb *jobBundle) {
+	defer func() {
```

Comment:
IMO better to crash the server to catch the bug

Reply:
crashing server means downtime, can't be good?

we can add metrics to discover job failure, plus logs to investigate.